proxy/envoy/mixer.go

```diff
@@ -59,22 +67,35 @@ const (
 	MixerForward = "mixer_forward"
 )
 
-// FilterMixerConfig definition
+// FilterMixerConfig definition.
+//
+// NOTE: all fields marked as DEPRECATED are part of the original v1
+// mixerclient configuration. They are deprecated and will be
+// eventually removed once proxies are updated.
+//
+// Going forwards all mixerclient configuration should represeted by
+// istio.io/api/mixer/v1/config/client/mixer_filter_config.proto and
+// encoded in the `V2` field below.
+//
 type FilterMixerConfig struct {
-	// MixerAttributes specifies the static list of attributes that are sent with
+	// DEPRECATED: MixerAttributes specifies the static list of attributes that are sent with
 	// each request to Mixer.
 	MixerAttributes map[string]string `json:"mixer_attributes,omitempty"`
 
-	// ForwardAttributes specifies the list of attribute keys and values that
+	// DEPRECATED: ForwardAttributes specifies the list of attribute keys and values that
 	// are forwarded as an HTTP header to the server side proxy
 	ForwardAttributes map[string]string `json:"forward_attributes,omitempty"`
 
-	// QuotaName specifies the name of the quota bucket to withdraw tokens from;
+	// DEPRECATED: QuotaName specifies the name of the quota bucket to withdraw tokens from;
 	// an empty name means no quota will be charged.
 	QuotaName string `json:"quota_name,omitempty"`
 
-	// If set to true, disables mixer check calls for TCP connections
+	// DEPRECATED: If set to true, disables mixer check calls for TCP connections
 	DisableTCPCheckCalls bool `json:"disable_tcp_check_calls,omitempty"`
+
+	// istio.io/api/mixer/v1/config/client configuration protobuf
+	// encoded as a generic map using canonical JSON encoding.
+	V2 map[string]interface{} `json:"v2,omitempty"`
```

Comment:
Please add comment
// if v2 field is not empty,  deprecated legacy fields are discarded.